rmf_traffic/test/unit/agv/test_Planner.cpp

```diff
@@ -458,6 +458,159 @@ SCENARIO("Test Options", "[options]")
     CHECK(*default_options.interrupt_flag());
   }
 
+  Planner::Options default_maxcostestimate(nullptr);
```

Comment:
```suggestion
  Planner::Options default_max_cost_estimate_options(nullptr);
```

I'd like to discourage run-on words. I think that reduces readability :sweat_smile:

Reply:
Sorry, I missed changing that one! I usually don't use them myself (this is why we have auto-complete, right?), but probably got lazy here...